migrations/Version202404141320101555_taoAdvancedSearch.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoAdvancedSearch\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\oatbox\reporting\Report;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoAdvancedSearch\scripts\install\RegisterTaskQueueServices;
+use oat\taoTaskQueue\model\Service\QueueAssociationService;
+use oat\taoTaskQueue\scripts\tools\BrokerFactory;
+
+final class Version202404141320101555_taoAdvancedSearch extends AbstractMigration
```

Comment:
Why are we creating a new migrations repeated as https://github.com/oat-sa/extension-tao-advanced-search/blob/develop/migrations/Version202106011320101488_taoAdvancedSearch.php#L25 ?

Why not just call this `RegisterTaskQueueServices` directly as CLI on prod whichever env this is broken?